### ISSUE TITLE
TabView: Setting SelectedIndex on init crashed on RS4 and below

### ISSUE THREAD

**Issue by teaP** [microsoft/microsoft-ui-xaml#812]
**Describe the bug**
Setting SelectedIndex on the TabView on init will crash on RS4 and below. (I think there must have been some bug fix in Selector that fixed this in later versions.)

**Steps to reproduce the bug**
<TabView SelectedIndex=0>
    <TabViewItem Icon="Home" Header="Home"/>
    <TabViewItem Icon="Shop" Header="Shop"/>
</TabView>

**Version Info**
This only happens on RS4 and below.

**Comment by michael-hawker** [comment]
Yeah, I've noticed this too.  There was definitely a change in RS4 that fixed this for ListView, ComboBox, etc...

I had filed an issue about the error message being really bad [here](https://github.com/dotnet/corefx/issues/35178) with .NET.
